tools/wptrunner/wptrunner/wptmanifest/serializer.py

```diff
@@ -99,8 +98,8 @@ def visit_StringNode(self, node):
             rv[0] += self.visit(child)[0]
         return rv
 
-    def visit_NumberNode(self, node):
-        return [ensure_text(node.data)]
+    def visit_NumberNode(self, node: NumberNode) -> List[str]:
```

Comment:
@jgraham are you familiar with this code? It seems to me the existing `ensure_text(node.data)` is broken and should throw a `TypeError` since `node.data` will be either `int` or `float`. Could this all be dead code? There are numbers in the manifest, but when I throw some exceptions in places that create `NumberNode` they seem unreached. That gives me very low confidence in these changes.